### DISCUSSION TITLE
mamba 405mk2 issues

### DISCUSSION THREAD

**Discussion by cryptictuna** [iNavFlight/inav#7547] [Q&A]
**hey guys   about to pull my last hairs out..!   PIcked up a Mamba 405mk2 from RDQ. came with betaflight.. I wanted to utilize Nav and ALT/POS hold etc..  It has a FURY4OSD (legacy) firmware from betaflight.. Ran into issues as also reported in other post loosing motor 3 when activating LED strip.. (marked LED2, as non of the other 3 pads work.) this is no good as I like the LEDs and should be able to use without loosing motor 3.. OK whatev.. So i flash over to Inav with "Fury4Osd" firmware.. BOOM leds work even with color, Motors work too using Inav output page.. NOW the issue is Frsky Receiver wont work. UGGGG. if not one thing its another.. Ive tried Everything. full. half duplex.. serial,, etc.. Nodda.. FLash back to betaflight , WOrks flawless cept no led without loosing motor3..  So then i dig into more posts.. another user suggested flashing INAV mamba405 firmware instead.. Hmmm.. ok. Done.. flashed successfully.. Receiver WORKS!!!!! but no LEDS ...  PLease help,, Maybe a new custom hex compile is needed to mash the 2 together... Im pretty frustrated at this point.. SO in summary..     

DEFAULT equipment Betalflight     NO GOOD, NO LEDS without LOOSING motor 3, NO NAV, NO IC2 Compass

1: diatone Mamba 405 Mk2  Defaults with Betaflight (FURYF4OSD legacy) firmware... 
2. BN-880 gps/compass
3. frsky XSR receiver
4. Taranis Q7 and Jumper t12
pros... 
1: SBUS receiver works flawless with smartport on TX1
2. GPS is seen
3. flight modes work. 
 
Cons
1. LEDS ON will wipe out motor 3..AND still no LEDS.. no motor3  no LEDS lights when enabled
2. . SDA SCL NOT working, NO Compass seen from the combo BN-880 with betaflight (tried all the timing hacks, etc. Nodda..  

FLASHED TO INAV 3.01 FURYf4OSD firmware    (NO GOOD, but closer, EVERYTHING WORKS EXCEPT SBUS,)

Pros.. 
1. Leds are now programmable. Color, blinks all good. 
2. ALL Motors work in testing mode Output page in Inav
3. GPS AND COMPASS BOTH SEEN AND OPERATIONAL and quickly too 
Con;
Cant get sbus to work at all.. Can get some random garbage when select spectrum 2048  with SBUS Serial  Nodda ZIP Zilch. with Sbus. so now dead in water again..  Start over

FLASHED TO iNAV 3.01 MAMBA405  (as per another posters suggestion)   UGGGGGGGGGGGGGGGG

Pros.. 
SBUS working immediately! smartport telemetry not recognized.
GPS and Compass Working.. 
LEDS, lol, 1 light of 6 on the strip is RED.. no programming or color changing, not working
Beeper is solid weird tone now.  

SO,, havent found a firmware that allows everything to work at once..  LEDS MOTORS NAV/Compass and receiver.  Id really prefer to use what the board is called FURYF4OSD inav firmware.. i was excited when i saw the leds and motor both work with inav FURY4OSD 3.01 But screamed at the wall when SBUS didnt.. I can fly with Mamba405 (MB405) firmware , receiver, GPS and compass do work. however isnt the right firmware for this board AND like in bf,   have no LEDS   what a headache.. I dont like spending all this time on the bench tweaking and reflashing, prgramming etc.. I want to fly already..

**Comment by SebastianKumor** [comment]
I have the mambaf405 target with inav 3.0.0 and got led sand servo on motor 3 output and everything works.
